src/Tester/BasicActivationTests.cs

```diff
@@ -15,36 +14,26 @@
 namespace UnitTests.General
 {
     [TestClass]
-    public class BasicActivationTests : UnitTestSiloHost
+    public class BasicActivationTests : UnitTestSiloHostEnsureDefaultStarted3
     {
-
-        public BasicActivationTests()
-            : base(new TestingSiloOptions { StartFreshOrleans = true })
-        {
-        }
-
-        [ClassCleanup]
-        public static void MyClassCleanup()
-        {
-            StopAllSilos();
-        }
-
         [TestMethod, TestCategory("Functional"), TestCategory("ActivateDeactivate"), TestCategory("GetGrain")]
         public void BasicActivation_ActivateAndUpdate()
         {
-            ITestGrain g1 = GrainClient.GrainFactory.GetGrain<ITestGrain>(1);
-            ITestGrain g2 = GrainClient.GrainFactory.GetGrain<ITestGrain>(2);
-            Assert.AreEqual(1L, g1.GetPrimaryKeyLong());
-            Assert.AreEqual(1L, g1.GetKey().Result);
-            Assert.AreEqual("1", g1.GetLabel().Result);
-            Assert.AreEqual(2L, g2.GetKey().Result);
-            Assert.AreEqual("2", g2.GetLabel().Result);
+            long g1Key = GetRandomGrainId();
+            long g2Key = GetRandomGrainId();
```

Comment:
yes, this test "should" work in both environments, as it's an improved version of the tests. Nevertheless it's more important now with the changes, as using fixed IDs is a bad idea if you don't restart the silos (there were a few tests trying to use the same grains and affecting each other). So even if I could improve the tests in a separate PR, it's something that we probably wouldn't just do if this other change wasn't coming too.